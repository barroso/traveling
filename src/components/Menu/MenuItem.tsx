import { Link } from "react-router-dom";

interface IProps {
  title: string;
  to: string;
}

const MenuItem = ({ title, to }: IProps) => {
  // const activeClass = "border-blue-500 border-b-4Ã¥";
  return (
    <Link
      to={to}
      className="py-4 px-2 text-blue-500 font-semibold">
      {title}
    </Link>
  );
};

export default MenuItem;
