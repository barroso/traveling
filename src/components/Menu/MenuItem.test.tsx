import MenuItem from './MenuItem';
//sample

describe('[components] - Cart', () => {
  const setup = () => {
    return (
        <MenuItem title='Test' to='/test'/>
    );
  };

  test('should render correctly', () => {
    const view = setup();
    expect(view).toMatchSnapshot();
  });
});