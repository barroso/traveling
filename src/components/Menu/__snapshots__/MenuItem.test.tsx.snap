// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[components] - Cart should render correctly 1`] = `
<MenuItem
  title="Test"
  to="/test"
/>
`;
